docker/Dockerfile

```diff
@@ -0,0 +1,59 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+
+FROM openjdk:8-jre-alpine
+MAINTAINER Alex Heneveld "alex@cloudsoft.io"
+
+RUN apk add --update bash openssh && rm -rf /var/cache/apk/*
+RUN adduser -D brooklyn
+WORKDIR /home/brooklyn
+
+# now install software brooklyn
+COPY target/files/ .
+RUN \
+  # remove option not relevan for java v8 \
+  sed -i.bk "s/ -XX:MaxPermSize=256m//" bin/brooklyn ; \
+  # install br for global use \
+  cp ./bin/br /usr/bin/br ; \
+  # and fix perms (irritating that docker won't do this) \
+  chown -R brooklyn:brooklyn .
+
+USER brooklyn
+
+# handle customisation, including passing args through to the container script and installing boms
```

Comment:
i thought that originally but the code for each project is basically the same, so it felt better to parametrise it here